cmake/modules/FindPythonPackage.cmake

```diff
@@ -0,0 +1,9 @@
+# - Find python package installation status
+
+macro(FIND_PYTHON_PKG PKG_NAME )
```

Comment:
How about an optional argument that controls the severity (warning vs. error)?

You've chosen to make these warnings. Why? packager and ASE configuration will fail without jsonschema. Wouldn't it be better to force the environment to be correct at config/build time instead of a more obscure message later?